backend/src/nodes/impl/pytorch/auto_split.py

```diff
@@ -12,13 +14,13 @@
 @torch.inference_mode()
 def pytorch_auto_split(
     img: np.ndarray,
-    model: torch.nn.Module,
+    model: ImageModelDescriptor,
     device: torch.device,
     use_fp16: bool,
     tiler: Tiler,
 ) -> np.ndarray:
     model = model.to(device)
-    model = model.half() if use_fp16 else model.float()
+    model.model.half() if use_fp16 else model.model.float()
```

Comment:
That's kinda cursed.

```suggestion
    if use_fp16:
        model.model.half()
    else:
        model.model.float()
```